### PR TITLE
Remove unused variables 'text_features' and 'image_features'

### PR DESCRIPTION
**Pull request by developer0hye**
The `text_features` and `image_features` was calculated but not used in the code.

### CODE REVIEW

**Comment by jongwook** [comment]
The lines are intended to show how to calculate the text and image features separately, as opposed to getting the logits which need both text and image inputs.
